qa/suites/rados/cephadm/smoke/fixed-2.yaml

```diff
@@ -20,6 +20,7 @@ roles:
   - prometheus.a
   - grafana.a
   - node-exporter.b
+  - iscsi.a
```

Comment:
```suggestion
  - iscsi.iscsi
```

Reply:
I think we need `ceph.iscsi.iscsi.a` here. Teuthology wants to control the daemons names.